jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -229,7 +233,36 @@ YUI.add('user-profile', function() {
             </span>
           </juju.components.UserProfileEntity>);
       });
-      return components;
+      return (
+        <div className="twelve-col no-margin-bottom">
+          <div className="user-profile__header twelve-col no-margin-bottom">
+            Models
+            <span className="user-profile__size">
+              ({envList.length})
+            </span>
+          </div>
+          <ul className="user-profile__list twelve-col">
+            <li className="user-profile__list-header twelve-col">
```

Comment:
These are nested as the parent ul has a border bottom and it needs to clear the floated child cols so it appears at the bottom instead of collapsing.